HEC.FDA.ViewModel/Storage/Connection.cs

```diff
@@ -275,7 +320,7 @@ public string[] TableNames()
         {
             return _SqliteReader.GetTableNames();
         }
-        public DatabaseManager.DataTableView GetTable(string TableName)
+        public object GetTable(string TableName)
```

Comment:
From what i can tell, this method is only ever used to check whether a table exists. Wrap this method in another method called "TableExists(string tableName)" and call this internally. It'll make it a lot clearer how it's used.

Reply:
done, makes it clearer